### DISCUSSION TITLE
What could be my first PR in here. Willing to contribute.

### DISCUSSION THREAD

**Discussion by KnockOutEZ** [pocketbase/pocketbase#2299] [Q&A]
Hey im a JS dev. Mostly worked with frontend but im also learning Go. Willing to contribute in this project as I liked pocketbase a lot. Which issue can be my first task? What should I start with. I followed the contribute,md and finished the setup. Could you suggest some stuff that I can work around? Would love something that involves both frontend and backend as im learning Go.

**Comment by ganigeorgiev** [comment]
Thank you but I don't need help with the available tasks from the [roadmap](https://github.com/orgs/pocketbase/projects/2) for now.
If you haven't already, please check the note in https://github.com/pocketbase/pocketbase#contributing.
